docs/markdown/intro/README.md

```diff
@@ -253,6 +255,24 @@ import { Scrollspy } from 'bootstrap-vue/es/directives'
 Vue.use(Scrollspy)
 ```
 
+Alternatively, you can import Plugins as follows:
+
+```js
+import BootstrapVue from 'bootstrap-vue'
+
+// This enables all the layout components such as <b-container>, <b-row>, <b-col>:
```

Comment:
I think we can remove the trailing `:` in the comments. We don't use them at other places in comments.